### PR TITLE
Updated Links

### PR DESCRIPTION
**Pull request by alyboyd**
Updated links, specifically links in document to bibliography.  Made some verbiage adjustments.  Corrected for accuracy.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/gdpr-arc-prof-services.md [hunk 1]
```diff
@@ -22,18 +22,18 @@ This accountability readiness checklist provides a convenient way to access info
...
+|***Privacy impact assessment (7.2.5)***|The customer should be aware of requirements for completing privacy impact assessments (when they should be performed, categories of data that might necessitate one, timing of completing the assessment).|Microsoft Professional Services provides guidance as to when and how to determine when to perform a DPIA, and an overview of the DPIA program at Microsoft including the involvement of the DPO, which is provided n the Service Trust Portal *Data* [Protection ,Impact Assessments (DPIAs) page](https://servicetrust.microsoft.com/ViewPage/GDPRDPIA).<p>For support for your DPIAs see:<br>- Key Information from Microsoft Professional Services for Customer Data Protection Impact Assessments [[9](gdpr-arc-prof-services.md#9)]</p>|Article (35)|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-prof-services.md]:
```suggestion
|***Privacy impact assessment (7.2.5)***|The customer should be aware of requirements for completing privacy impact assessments (when they should be performed, categories of data that might necessitate one, timing of completing the assessment).|Microsoft Professional Services provides guidance as to when and how to determine when to perform a DPIA, and an overview of the DPIA program at Microsoft including the involvement of the DPO, which is provided n the Service Trust Portal *Data* [Protection, Impact Assessments (DPIAs) page](https://servicetrust.microsoft.com/ViewPage/GDPRDPIA).<p>For support for your DPIAs see:<br>- Key Information from Microsoft Professional Services for Customer Data Protection Impact Assessments [[9](gdpr-arc-prof-services.md#9)]</p>|Article (35)|
```

File: microsoft-365/compliance/gdpr-arc-prof-services.md [hunk 2]
```diff
@@ -22,18 +22,18 @@ This accountability readiness checklist provides a convenient way to access info
...
 |***Contracts with PII Processors (7.2.6)***|The customer should ensure that their contracts with processors include requirements for aiding with any relevant legal or regulatory obligations related to processing and protecting personal data.|The Microsoft contracts that require us to aid with your obligations under the GDPR, including support for the data subject's rights.<br>- Microsoft Professional Services Data Protection Addendum [[1](gdpr-arc-prof-services.md#1)]|(5)(2), (28)(3)(e), (28)(9)|
 |***Records related to processing PII (7.2.7)***|The customer should maintain all necessary and required records related to processing personal data (for example, purpose, security measures, etc.). Where some of these records must be provided by a sub-processor, the customer should ensure that they can obtain such records.|Microsoft Professional Services maintains records necessary demonstrate compliance and support for accountability under the GDPR. See the Microsoft Professional Services Security Documentation [[2](gdpr-arc-prof-services.md#2)]|(5)(2), (24)(1), (30)(1)(a), (30)(1)(b), (30)(1)(c), (30)(1)(d), (30)(1)(g), (30)(1)(f), (30)(3), (30)(4), (30)(5)|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-prof-services.md]:
```suggestion
|***Records related to processing PII (7.2.7)***|The customer should maintain all necessary and required records related to processing personal data (for example, purpose, security measures, etc.). Where some of these records must be provided by a sub-processor, the customer should ensure that they can obtain such records.|Microsoft Professional Services maintains records necessary to demonstrate compliance and support for accountability under the GDPR. See the Microsoft Professional Services Security Documentation [[2](gdpr-arc-prof-services.md#2)]|(5)(2), (24)(1), (30)(1)(a), (30)(1)(b), (30)(1)(c), (30)(1)(d), (30)(1)(g), (30)(1)(f), (30)(3), (30)(4), (30)(5)|
```

File: microsoft-365/compliance/gdpr-arc-prof-services.md [hunk 3]
```diff
@@ -42,36 +42,36 @@ This control structure is also used to organize the presentation of the internal
...
+| ***Provide mechanism to modify or withdraw consent (7.3.4)*** | The customer should understand requirements for informing users about their right to access, correct, and/or erase their personal data and for providing a mechanism for which them to do so. If a third-party system is used and provides this mechanism as part of its functionality, the customer should utilize that functionality as necessary. | Information about capabilities in Microsoft services that you can use when defining the information you provide to data subjects when requesting consent.<br>- Microsoft Professional Services Data Subject Requests for the GDPR [[7](gdpr-arc-prof-services.md#7)] | (7)(3), (13)(2)(c), (14)(2)(d), (18)(1)(a), (18)(1)(b), (18)(1)(c), (18)(1)(d) |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-prof-services.md:48]:
```suggestion
| ***Provide mechanism to modify or withdraw consent (7.3.4)*** | The customer should understand requirements for informing users about their right to access, correct, and/or erase their personal data and for providing a mechanism for them to do so. If a third-party system is used and provides this mechanism as part of its functionality, the customer should utilize that functionality as necessary. | Information about capabilities in Microsoft services that you can use when defining the information you provide to data subjects when requesting consent.<br>- Microsoft Professional Services Data Subject Requests for the GDPR [[7](gdpr-arc-prof-services.md#7)] | (7)(3), (13)(2)(c), (14)(2)(d), (18)(1)(a), (18)(1)(b), (18)(1)(c), (18)(1)(d) |
```

File: microsoft-365/compliance/gdpr-arc-prof-services.md [hunk 4]
```diff
@@ -42,36 +42,36 @@ This control structure is also used to organize the presentation of the internal
...
+| ***Correction or erasure (7.3.7)*** | The customer should understand requirements for informing users about their right to access, correct, and/or erase their personal data and for providing a mechanism for which them to do so. If a third-party system is used and provides this mechanism as part of its functionality, the customer should utilize that functionality as necessary. | Information about Microsoft services relating to their ability to access, correct, or erase personal data that you can include in the data you provide to data subjects.<br>- Microsoft Professional Services Data Subject Requests for the GDPR [[7](gdpr-arc-prof-services.md#7)]||
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-prof-services.md]:
```suggestion
| ***Correction or erasure (7.3.7)*** | The customer should understand requirements for informing users about their right to access, correct, and/or erase their personal data and for providing a mechanism for them to do so. If a third-party system is used and provides this mechanism as part of its functionality, the customer should utilize that functionality as necessary. | Information about Microsoft services relating to their ability to access, correct, or erase personal data that you can include in the data you provide to data subjects.<br>- Microsoft Professional Services Data Subject Requests for the GDPR [[7](gdpr-arc-prof-services.md#7)]||
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@alyboyd - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by alyboyd** [comment]
This has been completed.  Please let me know if there is anything else needed on my end.

Thanks,
Aly

**Comment by mypil** [comment]
@robmazz - PR has been copy edited and is ready for final review. Please take note that this is a GDPR topic so I will leave it to your discretion whether to merge this PR or not. 

Thank you.

cc: @kenwith

**Comment by robmazz** [comment]
@alyboyd - thanks for the feedback, but this is more tightly controlled topic that requires review and approval by our legal department. We do not accept verbiage updates to this topic.

**Comment by alyboyd** [comment]
Hi Robert,

Please advise who should review these changes.  It was mainly changes to correct links that were inaccurate (links in document that point to links in the bibliography).  We have updated this before without further legal review.  Is this a new process?

Thanks,
Aly

**Pull request closed without merging**